draft-ietf-rats-corim.md

```diff
@@ -2466,11 +2464,11 @@ If any of the fields does not match, then the condition ECT does not match the A
 The Verifier SHALL compare each field which is present in the condition ECT `environment-map` against the corresponding field in the ACS entry `environment-map` using binary comparison.
 Before performing the binary comparison, the Verifier SHOULD convert both `environment-map` fields into a form which meets CBOR Core Deterministic Encoding Requirements {{-cbor}}.
 
-If all fields which are present in the condition ECT `environment-map` are present in the ACS entry and are binary identical, then the environments match.
+If all fields which are present in the condition ECT `environment-map` are present in the ACS entry and are equivalent, then the environments match.
```

Comment:
Should we not qualify what equivalent means?

Reply:
I think that we can revert the equivalence text for ECTs since there's already a translation to them.